benchmarks/time_setup_models_and_sims.py

```diff
@@ -89,12 +149,24 @@ def time_setup_SPMe_simulation(self, with_experiment):
 
 class TimeBuildDFNSimulation:
     # with_experiment
-    params = [False, True]
-
-    def __init__(self):
-        self.param = pybamm.ParameterValues("Marquis2019")
-
-    def time_setup_DFN_simulation(self, with_experiment):
+    params = (
+        [False, True],
+        [
+            "Marquis2019",
+            "ORegan2021",
+            "NCA_Kim2011",
+            "Prada2013",
+            "Ai2020",
+            "Ramadass2004",
+            "Mohtat2020",
+            "Chen2020",
+            "Chen2020_plating",
+            "Ecker2015",
+        ],
```

Comment:
Could you make a global variable with a list of these parameter sets outside the classes and use that in all the functions to reduce code duplication